sql/sql_connect.cc

```diff
@@ -1078,6 +1098,17 @@ static int check_connection(THD *thd)
     return 1; /* The error is set by alloc(). */
   }
 
+  if(check_require_secured_transport(thd))
+  {
+    Host_errors errors;
+    errors.m_ssl= 1;
+    inc_host_errors(thd->main_security_ctx.ip, &errors);
+    status_var_increment(thd->status_var.access_denied_errors);
```

Comment:
may be `thd->status_var.denied_connections` ? not sure

Reply:
I switched to denied_connections since it does make more sense

Reply:
`denied_connections` is a global var I believe, not a session var, so it would be `status_var_increment(denied_connections)`

Reply:
I put back the original part, should be good now